ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -352,16 +214,16 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'R1S Headmarker Line Stack',
+      id: 'R1S Headmarker Nailchipper Spread',
       type: 'HeadMarker',
-      netRegex: { id: headMarkerData.lineStack, capture: true },
+      netRegex: { id: headMarkerData.spreadMarker1, capture: false },
```

Comment:
```suggestion
      netRegex: { id: headMarkerData.spreadMarker1, capture: false },
      condition: Conditions.targetIsYou(),
```

I think a self-targeted condition would make sense here, because you don't technically need the warning when the other role group gets the headmarker.  If the other role group gets it first, you need to be in to bait the Quadruple (and can't spread); and if the other role group gets it second, you actually don't need to spread from them while they have it, since the vuln has fallen off and it can actually be taken with your partner.

Reply:
It could be useful to have some sort of indicator that you don't have it and need to bait first, before any role has baited yet.